### ISSUE TITLE
Source map issue

### ISSUE THREAD

**Issue by daanaerts** [alitaheri/jss-rtl#2]
Hi,

on line 31 of `./lib/main.js` there seems to be a problem with a sourceMap url

```
...
            }
            var flip = opt === 'out'; // If it's set to opt-out, then it should flip by default
            if (typeof sheet.options.flip === 'boolean') {
                flip = sheet.options.flip;
            }
            if (typeof style.flip === 'boolean') {
                flip = style.flip;
                delete style.flip;
            }
            if (!flip) {
                return style;
            }
            return convert(style);
        },
    };
}
exports.default = jssRTL;
//# sourceMappingURL=main.js.map <----- 
```

when running this through `source-map-loader` I get the following error:

```WARNING in ./node_modules/jss-rtl/lib/main.js
(Emitted value instead of an instance of Error) Cannot find source file '../src/main.ts': Error: Can't resolve '../src/main.ts' in '/Users/anonymous/workspace/sylc/webapp/node_modules/jss-rtl/lib'
NonErrorEmittedError: (Emitted value instead of an instance of Error) Cannot find source file '../src/main.ts': Error: Can't resolve '../src/main.ts' in '/Users/anonymous/workspace/sylc/webapp/node_modules/jss-rtl/lib'```

**Comment by daanaerts** [comment]
Second looks it's something with the TS setup. Sourcemap points to `src` map, but then when I run it in my project with `source-map-loader` it looks in `src` instead of `lib`

**Comment by alitaheri** [comment]
Ops, looks like I forgot to add the `src` folder to the package.json file 😅 😅 

Thanks @daanaerts  👍  👍

**Comment by alitaheri** [comment]
Fixed in `v0.2.1` 👍
